scripts/script.yaml

```diff
@@ -140,6 +141,13 @@
                   required: false
             - default: true                   # <---- for returning reporters, pass
 
+        - do:                                 # <---- location is the concatenation of an optional street and the city
```

Comment:
you can change this calculation to be conditional
see the alias calculation below, which calculates the alias only if alias is undefined

Reply:
done